commit 39a3d2d19cc8dc9d7ec3a1fefe95d7de0c6dc317
tree 4a0a50fb2eb84e83a1f6531f777def4a2f58eb8a
parent f0861f82d9976fab8624f056fa6880e6f420e89f
author Masakazu Mokuno <mokuno@sm.sony.co.jp> 1184920447 +0900
committer Jeff Garzik <jeff@garzik.org> 1185308920 -0400

    ps3: removed calling netif_poll_enable() in open()
    
    Removed use of netif_poll_enable() in open function.
    
    Signed-off-by: Masakazu Mokuno <mokuno@sm.sony.co.jp>
    Signed-off-by: Jeff Garzik <jeff@garzik.org>

---
 drivers/net/gelic_net.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/gelic_net.c	2007-08-27 14:01:21.000000000 -0400
+++ b/drivers/net/gelic_net.c	2007-08-27 14:03:56.000000000 -0400
@@ -1136,7 +1136,6 @@ static int gelic_net_open(struct net_dev
 
 	netif_start_queue(netdev);
 	netif_carrier_on(netdev);
-	netif_poll_enable(netdev);
 
 	return 0;
 
